src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackageExtractionContext.cs

```diff
@@ -17,10 +16,13 @@ public class PackageExtractionContext
         /// </summary>
         public bool UseLegacyPackageInstallPath { get; set; }
 
+        public PackageSaveModes PackageSaveMode { get; set; }
+
         public PackageExtractionContext()
         {
-            CopySatelliteFiles = false;
+            CopySatelliteFiles = true;
```

Comment:
I will use C#6 style code. It was a mistake that the value for CopySatelliteFiles was false by default. It should be true by default. I have added tests to test that existing codepaths work properly.